### PR TITLE
chore: rename blog post

### PR DESCRIPTION
**Pull request by Wompipomp**
### Description
For better SEO and description of the blog post, this PR renames the blog post to include the headline words in the URL. It also renames the corresponding image.

### CODE REVIEW

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-path: "/blog/scoped-registry/"
```
Comment (feedm3) [inline blog-posts/enterprises-benefit-from-scoped-npm-registries.md:2]: You can redirect the old link to the new one in the redirect.js file. This way, the old links will still work.

**Review by feedm3 — CHANGES_REQUESTED** [review]

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md (hunk 1 above)
Comment (Wompipomp) [reply blog-posts/enterprises-benefit-from-scoped-npm-registries.md:2]: cool thanks. Didn't know this. :raised_hands:

Comment (georgiee) [reply blog-posts/enterprises-benefit-from-scoped-npm-registries.md:2]: Maybe it's not that critical for short lived blog posts? I don't see the metaverse blog post in search engines yet neither do I see the npm blog post (which is maybe another issue?). I would skip that redirect here.

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+path: "/blog/enterprises-benefit-from-scoped-npm-registries/"
 date: "2022-01-17"
 title: "How enterprises benefit from scoped npm registries"
-featuredImage: images/npm-scoped-registry.jpg
+featuredImage: images/enterprises-benefit-from-scoped-npm-registries.jpg
```
Comment (georgiee) [inline blog-posts/enterprises-benefit-from-scoped-npm-registries.md:5]: images probably don't have any effect on SEO, changing the path is enough 🤓

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-path: "/blog/scoped-registry/"
+path: "/blog/enterprises-benefit-from-scoped-npm-registries/"
```
Comment (georgiee) [inline blog-posts/enterprises-benefit-from-scoped-npm-registries.md:2]: 👍

**Review by feedm3 — APPROVED** [review]

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md (hunk 1 above)
Comment (feedm3) [reply blog-posts/enterprises-benefit-from-scoped-npm-registries.md:2]: It's not critical, but it costs close to nothing to add it. I already had the blog post in my Feedly feed, so the link is already somewhere out there 😄

File: blog-posts/enterprises-benefit-from-scoped-npm-registries.md (hunk 2 above)
Comment (Wompipomp) [reply blog-posts/enterprises-benefit-from-scoped-npm-registries.md:5]: I know, I just wanted to clearly name it so everybody knows that it is from this blog post :)

**Pull request merged**